base/install_motor.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -ex
+
+git clone --depth 1 --branch ${MOTOR_VERSION} \
+    https://github.com/epics-modules/motor
+
+cd motor/modules
+
+git submodule update --init --depth 1 -j ${JOBS} \
+    motorMotorSim
+
+module_releases="
+MOTOR=${EPICS_MODULES_PATH}/motor
+MOTOR_MOTORSIM=${EPICS_MODULES_PATH}/motor/modules/motorMotorSim
+"
+echo "$module_releases" >> ${EPICS_MODULES_PATH}/../RELEASE
+
+cd ../configure
+
+echo "
+EPICS_BASE=${EPICS_BASE_PATH}
+MOTOR=${EPICS_MODULES_PATH}/motor
+ASYN=${EPICS_MODULES_PATH}/asyn
+SNCSEQ=${EPICS_MODULES_PATH}/seq
+AUTOSAVE=${EPICS_MODULES_PATH}/autosave
+BUSY=${EPICS_MODULES_PATH}/busy
+IPAC=${EPICS_MODULES_PATH}/ipac
+" > RELEASE.local
+
+ln -s RELEASE.local RELEASE_PRODS.local
+ln -s RELEASE.local RELEASE_LIBS.local
```

Comment:
This is a very areaDetector specific way of configuring things. It seems that [Motor](https://github.com/epics-modules/motor/tree/d1ed86f128733070ec720286d1e1fad05e8e8156/configure) does not need to define neither `RELEASE_PRODS.local` nor `RELEASE_LIBS.local`.

(TBH, I think even areaDetector should not need them. That's why I have symlinked them.)